src/neptune/metadata_containers/metadata_container.py

```diff
@@ -643,7 +645,7 @@ def get_url(self) -> str:
 
     def _startup(self, debug_mode):
         if not debug_mode:
-            logger.info(self.get_url())
+            logger.info(f"Neptune initialized. Monitor the logging in the app: {self.get_url()}")
```

Comment:
@normandy7 and another one

Reply:
If it's the same in read-only mode, let's not mention logging.

```suggestion
            logger.info(f"Neptune initialized. View the run in the app: {self.get_url()}")
```

Reply:
Yes, this needs to be generic -> applying